UnitsNet/UnitDefinitions/Energy.json

```diff
@@ -89,6 +89,57 @@
           "Abbreviations": [ "Wh" ]
         }
       ]
+    },
+    {
+      "SingularName": "EuTherm",
```

Comment:
This is called EC Therms in the Wikipedia article, probably because the unit is from the time when EU was called EC. Is EU Therm the most common name right now?

Reply:
Well, I wish I could tell. From my experience people just use **therm** implicitly meaning a US or EU unit, that they solely use everywhere in documentation across a project. So looks like it is up to us how to call it.

Reply:
I would go with calling it EC therm then to match Wikipedia. Then we are consistent with at least one other source. I'll leave the last word to @anjdreas